elytron/src/test/java/org/wildfly/extension/elytron/KeyStoresTestCase.java

```diff
@@ -1037,35 +1038,35 @@ public void testObtainCertificateWithoutAgreeingToTermsOfService() throws Except
     @Test
     public void testObtainCertificateWithKeySize() throws Exception {
         addKeyStore(ACCOUNTS_KEYSTORE, ACCOUNTS_KEYSTORE_NAME, ACCOUNTS_KEYSTORE_PASSWORD);
-        addCertificateAuthorityAccount("account6");
+        addCertificateAuthorityAccount("account6v2", "mailto:admin@myexample.com");
         addKeyStore(); // to store the obtained certificate
         server = setupTestObtainCertificateWithKeySize();
         String alias = "server";
         String keyAlgorithmName = "RSA";
         int keySize = 4096;
         KeyStore keyStore = getKeyStore(KEYSTORE_NAME);
         assertFalse(keyStore.containsAlias(alias));
-        obtainCertificate(keyAlgorithmName, keySize, "inlneseppwkfwew.com", alias, keyStore);
+        obtainCertificate(keyAlgorithmName, keySize, "inlneseppwkfwewv2.com", alias, keyStore);
     }
 
     @Test
     public void testObtainCertificateWithECPublicKey() throws Exception {
         addKeyStore(ACCOUNTS_KEYSTORE, ACCOUNTS_KEYSTORE_NAME, ACCOUNTS_KEYSTORE_PASSWORD);
-        addCertificateAuthorityAccount("account7");
+        addCertificateAuthorityAccount("account7v2", "mailto:admin@myexample.com");
         addKeyStore(); // to store the obtained certificate
         server = setupTestObtainCertificateWithECPublicKey();
         String alias = "server";
         String keyAlgorithmName = "EC";
         int keySize = 256;
         KeyStore keyStore = getKeyStore(KEYSTORE_NAME);
         assertFalse(keyStore.containsAlias(alias));
-        obtainCertificate(keyAlgorithmName, keySize, "mndelkdnbcilohg.com", alias, keyStore);
+        obtainCertificate(keyAlgorithmName, keySize, "mndelkdnbcilohgv2.com", alias, keyStore);
     }
 
     @Test
     public void testObtainCertificateWithUnsupportedPublicKey() throws Exception {
         addKeyStore(ACCOUNTS_KEYSTORE, ACCOUNTS_KEYSTORE_NAME, ACCOUNTS_KEYSTORE_PASSWORD);
-        addCertificateAuthorityAccount("account7");
+        addCertificateAuthorityAccount("account7v2", "mailto:admin@anexample.com");
```

Comment:
Just noticed that the contact url used for "account7v2" is different in the previous test. Is there a reason for this?

Reply:
Sorry I missed this as the test was mean to fail, updated to the correct email